app/models/user.js

```diff
@@ -3,15 +3,15 @@ module.exports = (sequelize, DataTypes) => {
     'users',
     {
       id: {
-        type: DataTypes.UUID,
-        defaultValue: sequelize.UUIDV4,
+        type: DataTypes.INTEGER,
+        autoIncrement: true,
         primaryKey: true
       },
       name: {
         type: DataTypes.STRING,
         allowNull: false
       },
-      lastName: {
+      last_name: {
```

Comment:
This change to `snake_case` should only be done in the migration, here in the model you only have to add an option so that it only takes care of matching the property in `camelCase` with the one in the DB that is in `snake_case`.